### PR TITLE
fix typos in "best practices" pages

### PR DESCRIPTION
**Pull request by thapasusheel**
This PR fix the typo that has been seen in various files.

```pseudo
distrbution > distribution
practies > practise
managment > management
unque > unique
configuraiton > configuration
```

### CODE REVIEW

File: docs/generated/actions/appstore.md [hunk 1]
```diff
@@ -454,7 +454,7 @@ fastlane deliver submit_build --build_number 830 --submission_information "{\"ex
 
 ### App Privacy Details
 
-Starting on December 8, 2020, Apple announced that developers are required to provide app privacy details that will help users understand an app's privacy practies. _deliver_ does not allow for updating of this information but this can be done with the _upload_app_privacy_details_to_app_store_ action. More information on [Uploading App Privacy Details](https://docs.fastlane.tools/uploading-app-privacy-details)
+Starting on December 8, 2020, Apple announced that developers are required to provide app privacy details that will help users understand an app's privacy practise. _deliver_ does not allow for updating of this information but this can be done with the _upload_app_privacy_details_to_app_store_ action. More information on [Uploading App Privacy Details](https://docs.fastlane.tools/uploading-app-privacy-details)
```
Comment (rogerluan) [inline docs/generated/actions/appstore.md:457]: This has to be modified in https://github.com/fastlane/fastlane instead as it's autogenerated code 🙏

Comment (rogerluan) [reply docs/generated/actions/appstore.md:457]: And I think the expected word is actually `practices` :)

File: docs/generated/actions/deliver.md [hunk 2]
```diff
@@ -454,7 +454,7 @@ fastlane deliver submit_build --build_number 830 --submission_information "{\"ex
 
 ### App Privacy Details
 
-Starting on December 8, 2020, Apple announced that developers are required to provide app privacy details that will help users understand an app's privacy practies. _deliver_ does not allow for updating of this information but this can be done with the _upload_app_privacy_details_to_app_store_ action. More information on [Uploading App Privacy Details](https://docs.fastlane.tools/uploading-app-privacy-details)
+Starting on December 8, 2020, Apple announced that developers are required to provide app privacy details that will help users understand an app's privacy practise. _deliver_ does not allow for updating of this information but this can be done with the _upload_app_privacy_details_to_app_store_ action. More information on [Uploading App Privacy Details](https://docs.fastlane.tools/uploading-app-privacy-details)
```
Comment (rogerluan) [inline docs/generated/actions/deliver.md:457]: Same

File: docs/generated/available-plugins.md [hunk 3]
```diff
@@ -2763,7 +2763,7 @@ Auto generated, please only modify https://github.com/fastlane/fastlane/blob/mas
 
 <p class="via-text">via Jiří Otáhal</p>
 
-> Automated version managment and generator of release notes.
+> Automated version management and generator of release notes.
```
Comment (rogerluan) [inline docs/generated/available-plugins.md:2766]: Same for this and all files under `/generated/` and `/plugins/` directories

**Comment by rogerluan** [comment]
Thanks for noticing these and taking the initiative to fix them @thapasusheel ! 🤗 

Out of the changes you made, all of them except 1 belong to the main fastlane repo, could you adjust this PR? I'll fix the typos in the main repo meanwhile :)

File: docs/generated/available-plugins.md [hunk 4]
```diff
@@ -7670,7 +7670,7 @@ Auto generated, please only modify https://github.com/fastlane/fastlane/blob/mas
 
 <p class="via-text">via William Lauze</p>
 
-> Fork for: Automated version managment and generator of release notes.
+> Fork for: Automated version management and generator of release notes.
```
Comment (rogerluan) [inline docs/generated/available-plugins.md:7673]: These references are pulled from 3rd party repositories. One repo made this typo, and 4 other repos forked it so the typo spread 😂 

Ref.: https://docs.fastlane.tools/plugins/available-plugins/

We'll have to revert these as this file gets autogenerated on every deploy 😅

File: docs/plugins/available-plugins.md [hunk 5]
```diff
@@ -23627,7 +23627,7 @@ Auto generated, please only modify https://github.com/fastlane/fastlane/blob/mas
 
 <p class="via-text">via ivan kat</p>
 
-> Sauce Labs android & ios configuraiton
+> Sauce Labs android & ios configuration
```
Comment (rogerluan) [inline docs/plugins/available-plugins.md:23630]: Same

File: docs/generated/available-plugins.md [hunk 6]
```diff
@@ -21597,7 +21598,7 @@ Auto generated, please only modify https://github.com/fastlane/fastlane/blob/mas
 
 <p class="via-text">via icyleaf</p>
 
-> Automatic generate app build number unque and human readable friendly, like yymmHHMM. both support iOS and Android.
+> Automatic generate app build number unique and human readable friendly, like yymmHHMM. both support iOS and Android.
```
Comment (rogerluan) [inline docs/generated/available-plugins.md:21601]: Same

**Comment by rogerluan** [comment]
I addressed all I could here: https://github.com/fastlane/fastlane/pull/21589

**Comment by thapasusheel** [comment]
@rogerluan Can you review it again?

**Review by rogerluan — CHANGES_REQUESTED** [review]
@thapasusheel not sure if you pushed any changes? The PR looks the same to me 🤔 

To be clear, only the changes made to `codemagic.md` are to be kept, the others should be reverted as they're autogenerated 🙏 

Thanks in advance!

**Comment by thapasusheel** [comment]
@rogerluan  I have updated as per you have said. Can you review it again?

File: docs/best-practices/xcodebuild-formatters.md [hunk 7]
```diff
@@ -26,7 +26,7 @@ scan(
   xcodebuild_formatter: "xcpretty"
 )
 
-# Specificy a local install of xcbeautify
+# Specifify a local install of xcbeautify
```
Comment (rogerluan) [inline docs/best-practices/xcodebuild-formatters.md:29]:
```suggestion
# Specify a local installation of xcbeautify
```

(Typo + I think installation is either more correct or simply sounds better)

**Review by rogerluan — CHANGES_REQUESTED** [review]

**Review by rogerluan — APPROVED** [review]
💪 thanks for your contributions @thapasusheel !

**Pull request merged**